README.md

```diff
@@ -1,2 +1,49 @@
 # lev-api-stress
-Stress testing for the `lev-api`.
+Stress testing for the [LEV API], based on the [Artillery.io] tool.
+
+
+## Running the test
+The performance tests can be started with the following command:
+```shell script
+./entrypoint.sh <path-to-config-file>
+```
+
+
+## Configuration
+There are two parts to the configuration:
+ - `./high-load.yml` - the main config file for [Artillery.io], containing:
+   - basic test setup
+   - security details
```

Comment:
Should this be authentication rather than security?

Reply:
Fixed.